src/libultra/libc/xldtob.c

```diff
@@ -4,8 +4,8 @@
 
 #define BUFF_LEN 0x20
 
-short _Ldunscale(short*, _Pft*);
-void _Genld(_Pft*, char, char*, short, short);
+static short _Ldunscale(short*, _Pft*);
```

Comment:
Personally I don't mind but we've avoided static so far for a more complete map file. Since it's needed for -O3 matching, maybe it would be nice to use a `STATIC` macro so it can be disabled?

Reply:
I think it'll be okay for these functions, so leaving as `static` for now